packages/cli/src/UserManagement/email/UserManagementMailer.ts

```diff
@@ -83,48 +95,111 @@ export class UserManagementMailer {
 	}
 
 	async notifyWorkflowShared({
-		recipientEmails,
-		workflowName,
-		baseUrl,
+		sharer,
+		newShareeIds,
 		workflowId,
-		sharerFirstName,
+		workflowName,
```

Comment:
we could also pass in `workflow` instead of `workflowId` and `workflowName` (just like what you did with `sharer`).